G4_DailyReport/src/main/java/com/example/G4_DailyReport/controller/manager/ProjectMemberController.java

```diff
@@ -0,0 +1,74 @@
+package com.example.G4_DailyReport.controller.manager;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.example.G4_DailyReport.model.Project;
+import com.example.G4_DailyReport.model.User;
+import com.example.G4_DailyReport.service.ProjectMemberService;
+import com.example.G4_DailyReport.util.CurrentUserUtil;
+
+import lombok.RequiredArgsConstructor;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/manager")
```

Comment:
```suggestion
@RequestMapping("/manager/project-management")
```
ri luôn để đỡ lặp code bên dưới